### PR TITLE
Chore/terraform

### PR DESCRIPTION
**Pull request by junminahn**
- Keyclaok configuration update
- Add build layers when building Keycloak image
- Add documentation for Keycloak docker directory

### CODE REVIEW

File: .github/workflows/publish-image-keycloak.yml [hunk 1]
```diff
@@ -55,3 +66,13 @@ jobs:
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
           file: docker/keycloak/Dockerfile-${{ startsWith(github.ref, 'refs/tags/7.4-37') && '7.4-37' || '7.5-9' }}
+          cache-from: type=local,src=/tmp/.buildx-cache
+          cache-to: type=local,dest=/tmp/.buildx-cache-new
+
+        # Temp fix
```
Comment (thegentlemanphysicist) [inline .github/workflows/publish-image-keycloak.yml:72]: Is this comment supposed to be hear?  Do we have a task in the backlog to make the 'temp fix' permanent?

File: docker/keycloak/configuration/standalone-openshift-7.5-9.xml [hunk 2]
```diff
@@ -97,13 +97,13 @@
                     <named-formatter name="OPENSHIFT" />
                 </formatter>
             </console-handler>
-            <!-- <periodic-rotating-file-handler name="FILE">
+            <periodic-rotating-file-handler name="AUDIT">
```
Comment (thegentlemanphysicist) [inline docker/keycloak/configuration/standalone-openshift-7.5-9.xml:100]: Out of curiosity, why was this commented out before? Was the rotatiting file handler feature broken?

File: localdev/keycloak-postgres.yml [hunk 3]
```diff
@@ -24,8 +26,10 @@ services:
...
       SSO_ADMIN_PASSWORD: admin
       TZ: America/Vancouver
-      JAVA_OPTS_APPEND: '-Djboss.persistent.log.dir=/'
+      JAVA_OPTS_APPEND: '-Djboss.persistent.log.dir=/opt/eap'
     ports:
       - 8080:8080
     depends_on:
       - postgres
+    volumes:
+      - log_data:/opt/eap:rw
```
Comment (thegentlemanphysicist) [inline localdev/keycloak-postgres.yml:35]: does the :rw set the read write permissions on the volume? I'd not seen that before

File: docker/keycloak/configuration/standalone-openshift-7.5-9.xml (hunk 2 above)
Comment (junminahn) [reply docker/keycloak/configuration/standalone-openshift-7.5-9.xml:100]: it was commented to work in local dev, but found a way to make it work in local dev by configuring the volume.

File: .github/workflows/publish-image-keycloak.yml (hunk 1 above)
Comment (junminahn) [reply .github/workflows/publish-image-keycloak.yml:72]: good point! actually the workaround is brought from the action's official team and there is an open PR for that.
https://github.com/docker/build-push-action/blob/master/docs/advanced/cache.md#local-cache

**Review by thegentlemanphysicist — APPROVED** [review]

**Pull request merged**